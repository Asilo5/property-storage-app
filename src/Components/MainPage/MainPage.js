import React from 'react';
import './MainPage.scss';

const MainPage = () => {
    return (
      <section>
          <header></header>
          <section className='looking-for-home'>
              <h2>Looking for a Home?..</h2>
              <p>but can't keep track of what you like</p>
          </section>
          <section className='small-intro'>
              <section>
                  <img src='https://images.unsplash.com/photo-1504615755583-2916b52192a3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60' alt='red brick house' />
                  <img src='https://images.unsplash.com/photo-1499916078039-922301b0eb9b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1100&q=80' alt='inside a white bedroom facing the window' />
              </section>
              <section>
                  <h2>Proplr</h2>
                  <p>Helps you store properties from multiple sites in one spot!</p>
                  <p>Just that easy</p>
              </section>
          </section>
          <footer>

          </footer>
      </section>
    )
}

export default MainPage;