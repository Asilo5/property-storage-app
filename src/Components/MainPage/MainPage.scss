header {
    background-image: url('https://images.unsplash.com/photo-1464146072230-91cabc968266?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60');
    background-size: cover;
    height: 40em;
    background-position: center;
}

.looking-for-home {
    background: white;
    padding: 20px;
    width: 50%;
    margin: auto;
    position: relative;
    top: -37px;
    border: 2px solid green;
    border-radius: 10px;
}

.small-intro {
    background-color: rgba(152, 169, 88, 0.685);
    display: flex;
    justify-content: center;
    padding: 5em;
    margin-top: -74px;
    max-height: 35em;
    img {
        height: 100%;
    }
    .bedroom-img {
        position: relative;
        top: 10%;
        right: 20%;
        border-bottom: double;
        border-right: double;
        padding: 0 5px 5px 0;
    }
    .brickhouse-img {
      padding: 5px 0 0 5px;
      border-top: double;
      border-left: double;
    }
}

footer {
    background-color: #436B38;
    padding: 20px;
}