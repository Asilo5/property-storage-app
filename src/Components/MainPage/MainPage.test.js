import React from 'react';
import { shallow } from 'enzyme';
import MainPage from './MainPage';

describe('MainPage', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<MainPage />)
    })

    it('should match snapshot with all information passing in correctly', () => {
        expect(wrapper).toMatchSnapshot();
    })
})