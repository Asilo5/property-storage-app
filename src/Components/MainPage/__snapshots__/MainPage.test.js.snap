// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage should match snapshot with all information passing in correctly 1`] = `
<section>
  <header />
  <section
    className="looking-for-home"
  >
    <h2>
      Looking for a Home?..
    </h2>
    <p>
      but can't keep track of what you like
    </p>
  </section>
  <section
    className="small-intro-section"
  >
    <section>
      <img
        alt="red brick house"
        className="brickhouse-img"
        src="https://images.unsplash.com/photo-1504615755583-2916b52192a3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60"
      />
      <img
        alt="inside a white bedroom facing the window"
        className="bedroom-img"
        src="https://images.unsplash.com/photo-1499916078039-922301b0eb9b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1100&q=80"
      />
    </section>
    <section
      className="intro"
    >
      <h2>
        Proplr
      </h2>
      <p>
        Helps you store properties from multiple sites in one spot!
      </p>
      <p>
        Just that easy.
      </p>
      <button>
        Get Started
      </button>
    </section>
  </section>
  <footer>
    <p>
      Created by Â© The Ginger Team
    </p>
  </footer>
</section>
`;
